api-reference/_hidden/GridBaseColumn/fixedPosition.md

```diff
@@ -5,8 +5,19 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the UI component's edge to which the column is fixed. Applies only if **columns[]**.[fixed](/api-reference/_hidden/GridBaseColumn/fixed.md '{basewidgetpath}/Configuration/columns/#fixed') is **true**.
+Specifies position in which the column is fixed. Applies only if **columns[]**.[fixed](/api-reference/_hidden/GridBaseColumn/fixed.md '{basewidgetpath}/Configuration/columns/#fixed') is **true**.
```

Comment:
Since the second sentence mentions "fixed = true", I think we can remove that part in the first sentence. This way, I can avoid the passive and the word "which". 

```suggestion
Specifies the column position. Applies only if **columns[]**.[fixed](/api-reference/_hidden/GridBaseColumn/fixed.md '{basewidgetpath}/Configuration/columns/#fixed') is **true**.
```